packages/orbit-components/src/primitives/MobileDialogPrimitive/index.js

```diff
@@ -17,6 +17,7 @@ const MobileDialog = ({
   content,
   stopPropagation = false,
   removeUnderlinedText,
+  fullWidth = false,
```

Comment:
why it should be `false` by default? it will not have any difference with `undefined`

Reply:
Almost, but this way we're being explicit. E.g. if at some point we decide that TooltipPrimitive's  `fullWidth` defaults to `true`, that won't break Tooltip.